pkgdb2/lib/__init__.py

```diff
@@ -2485,7 +2485,9 @@ def edit_action_status(
             raise PkgdbException(
                 'You are not allowed to edit this request')
     elif action_status in ['Awaiting Review', 'Blocked']:
-        if not pkg_admin and not pkgdb_admin:
+        # Requester can re-set 'request.unretire' to 'Awaiting Review'
+        if (not pkg_admin and not pkgdb_admin and
+                requester and admin_action.action != 'request.unretire'):
```

Comment:
This looks potentially problematic to me.

Is it possible for ``pkgdb_admin`` to be True and ``requester`` to be false?

Does this need to look like this instead:

```python
if (not pkg_admin and not pkgdb_admin and (requester and admin_action != 'request.unretire))
```

The more I look at it, the more confused I get.  :p  (It might be okay as is).